core/common/src/geometry/ElementGeometry.ts

```diff
@@ -136,6 +136,101 @@ export interface ElementGeometryUpdate {
   viewIndependent?: boolean;
 }
 
+/** Values for [[BRepGeometryCreate.operation]]
+ * @alpha
+ */
+export enum BRepGeometryOperation {
+  /** Unite target (first entry) with one or more tool entities. */
+  Unite = 0,
+  /** Subtract one or more tool entities from target entity (first entry) */
+  Subtract = 1,
+  /** Intersect target (first entry) with one or more tool entities */
+  Intersect = 2,
+  /** Sew the given set of surfaces together by joining those that share edges in common */
+  Sew = 3,
+  /** Create a cut in the target (first entry) using a planar region (second entry) and optional depth */
+  Cut = 4,
+  /** Create a pad or pocket in the target (first entry) using a planar region (second entry) */
+  Emboss = 5,
+  /** Create a solid from a surface by offsetting using the specified forward and backward distances */
+  Thicken = 6,
+  /** Create a shelled solid by offsetting all faces by the supplied distance */
+  Hollow = 7,
+  /** Create a solid or surface by sweeping a planar profile (first entry) along a path (second entry) */
+  Sweep = 8,
+  /** Create a solid or surface by lofting through a set of paths or regions */
+  Loft = 9,
+  /** Create a solid or sheet with all non-smooth/non-laminar edges rounded */
+  Round = 10,
+}
+
+/** Parameters for Cut operation
+ * @alpha
+ */
+export interface BRepCutProps {
+  /** Optional cut depth; through solid if not specified */
+  distance?: number;
+  /** Optional cut direction; in direction of profile normal if not specified */
+  bothDirections?: boolean;
+}
+
+/** Parameters for Thicken operation
+ * @alpha
+ */
+export interface BRepThickenProps {
+  /** Offset to apply in surface normal direction; at least one of frontDistance or backDistance must be specified */
+  frontDistance?: number;
+  /** Offset to apply in reverse surface normal direction */
+  backDistance?: number;
+}
+
+/** Parameters for Hollow operation
+ * @alpha
+ */
+export interface BRepHollowProps {
+  /** Offset to apply to all faces; positive value shells outwards, negative value shells inwards */
+  distance: number;
+}
+
+/** Parameters for Round operation
+ * @alpha
+ */
+export interface BRepRoundProps {
+  /** Radius to apply to all non-laminar/non-smooth edges */
+  radius: number;
+}
+
+/** Parameters for [IModelDb.createBRepGeometry]($imodeljs-backend)
+ * @alpha
+ */
+export interface BRepGeometryCreate {
+  /** The operation to perform on the input geometry */
+  operation: BRepGeometryOperation;
+  /** The input target and tool geometry (curves, surfaces, and solids) to perform the given operation on */
+  entryArray: ElementGeometryDataEntry[];
+  /** A function to call for the result geometry */
+  onResult: BRepGeometryFunction;
+  /** Whether disjoint results should be returned as multiple entries or a single disjoint body */
+  separateDisjoint?: boolean;
+  /** Options and parameters for this operation */
+  parameters?: any;

```

Comment:
Expected parameters for operatoins documented anywhere?

Reply:
Yes, there are interfaces like BRepHollowProps in same file.